src/c++/vdo/fake/asm/unaligned.h

```diff
@@ -1,17 +1,12 @@
-/*
- * FOR INTERNAL USE ONLY, DO NOT DISTRIBUTE!!!!
+/* SPDX-License-Identifier: GPL-2.0-only */
  *
  * Unit test implementation of asm/unaligned.h
  *
```

Comment:
Huh. So this file confused me because it should conflict with the uds asm/unaligned.h header. But it looks like nothing in VDO uses the two macros defined here, so we could probably remove the file entirely. Since that likely entails some manifest changes, etc., I'd make a separate commit before this one to remove the file and adjust anything else that needs adjusting.

Reply:
Added a commit to remove this. Our MANIFEST only specifies the c++/uds/userLinux/uds/asm/unaligned.h, so no changes were needed.